nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -91,6 +92,7 @@
     externalInitAnnotations = getFlagStringSet(flags, FL_EXTERNAL_INIT_ANNOT);
     isExhaustiveOverride = flags.getBoolean(FL_EXHAUSTIVE_OVERRIDE).orElse(false);
     isSuggestSuppressions = flags.getBoolean(FL_SUGGEST_SUPPRESSIONS).orElse(false);
+    treatGeneratedAsUnannotated = flags.getBoolean(FL_GENERATED_UNANNOTATED).orElse(false);
```

Comment:
Wouldn't this be better as a string set of `Generated`-like annotations? It is fine for now, but we will probably eventually run into some tool that uses a different annotation for its generated code.

Reply:
Agreed.  But in the spirit of limiting complexity I'm going to leave this for now as I think it's the most common case.  If we need to add a more general option later we can still keep this one for backward compatibility.